analyzer/src/main/kotlin/managers/Pub.kt

```diff
@@ -48,20 +58,42 @@ import org.ossreviewtoolkit.model.config.RepositoryConfiguration
 import org.ossreviewtoolkit.model.createAndLogIssue
 import org.ossreviewtoolkit.model.yamlMapper
 import org.ossreviewtoolkit.utils.CommandLineTool
+import org.ossreviewtoolkit.utils.ORT_NAME
+import org.ossreviewtoolkit.utils.OkHttpClientHelper
 import org.ossreviewtoolkit.utils.Os
 import org.ossreviewtoolkit.utils.ProcessCapture
 import org.ossreviewtoolkit.utils.collectMessagesAsString
 import org.ossreviewtoolkit.utils.getPathFromEnvironment
 import org.ossreviewtoolkit.utils.isSymbolicLink
 import org.ossreviewtoolkit.utils.log
+import org.ossreviewtoolkit.utils.ortToolsDirectory
 import org.ossreviewtoolkit.utils.showStackTrace
 import org.ossreviewtoolkit.utils.textValueOrEmpty
+import org.ossreviewtoolkit.utils.unpack
 
 private const val GRADLE_VERSION = "5.6.4"
 private const val PUB_LOCK_FILE = "pubspec.lock"
 
+private val flutterCommand = if (Os.isWindows) "flutter.bat" else "flutter"
+private val pubCommand = if (Os.isWindows) "pub.bat" else "pub"
+
+private val flutterVersion = Os.env["FLUTTER_VERSION"] ?: "v1.12.13+hotfix.9-stable"
```

Comment:
I see this still uses Flutter  v1.12.13 which current stable is 2.0.3 see https://flutter.dev/docs/development/tools/sdk/releases so we should upgrade else scans will likely fail as the Flutter community is using more and more 2.x.

Reply:
Yes, the intention of this PR was not to upgrade Flutter, but to implement the bootstrap mechanism without any further changes to keep things as simple as possible. We can do the upgrade as a follow-up.